packages/components/scss/BaseStyleSheet.scss

```diff
@@ -3,6 +3,11 @@
 @import './custom.scss';
 @import 'bootstrap/scss/bootstrap';
 
+:root {
+  // TODO: where's a better place to put this?
+  --dh-svg-inline-icon-size: 16px;
```

Comment:
This seems fine to me for now. Possible we could move it to the module containing the inline SVG variables once my PR is done, but there's not really a better spot for now